yazbyz/week04/BJ_9663.java

```diff
@@ -0,0 +1,52 @@
+//BaekJoon9663
+//author : Hyejin Eom
+//20210318
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class BJ_9663 {
+	public static int[] arr;
+	public static int N;
+	public static int count = 0;
+	public static void main(String[] args) throws IOException {
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		N = Integer.parseInt(br.readLine());
+		arr = new int[N];
+		nQueen(0);
+		System.out.println(count);
+	}
+	public static void nQueen(int depth) {
+		// 모든 원소를 다 채운 상태면 count 증가 및 return
+		if (depth == N) {
+			count++;
+			return;
+		}
+		for (int i = 0; i < N; i++) {
+			arr[depth] = i;
+			// 놓을 수 있는 위치일 경우 재귀호출
+			if (Possibility(depth)) {
+				nQueen(depth + 1);
+			}
+		}
+	}
+	public static boolean Possibility(int col) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.